charts/fluent-operator/values.yaml

```diff
@@ -63,7 +63,24 @@ fluentbit:
       enable: false
       host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
       port: 9200
+      path: ""
```

Comment:
We'd better comment the new added content because there're not required by default. User can remove the comments when they need this
```yaml
  output:
    es:
      enable: false
      host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
      port: 9200
      logstashPrefix: ks-logstash-log
#      path: ""
#      bufferSize: "4KB"
#      index: "fluent-bit"
#      httpUser:
#     httpPassword:
#      logstashFormat: true
#      replaceDots: false
#     enableTLS: false
#      tls:
#        verify: On
#       debug: 1
#        caFile: "<Absolute path to CA certificate file>"
#        caPath: "<Absolute path to scan for certificate files>"
#        crtFile: "<Absolute path to private Key file>"
#        keyFile: "<Absolute path to private Key file>"
#        keyPassword:
#        vhost: "<Hostname to be used for TLS SNI extension>"
```

Reply:
Done.